test/components/Signup/PodRegistrationForm.test.jsx

```diff
@@ -0,0 +1,28 @@
+// React imports
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+// Testing imports
+import { expect, it, describe } from 'vitest';
+
+// Component being tested
+import PodRegistrationForm from '../../../src/components/Signup/PodRegistrationForm';
```

Comment:
Non-issue: you can use `@components/Signup/PodRegistrationForm`. You don't need to use relative pathing.